fe1-web/model/objects/RollCall.ts

```diff
@@ -116,6 +122,8 @@ export class RollCall implements LaoEvent {
     const obj: any = JSON.parse(JSON.stringify(this));
     return {
       ...obj,
+      start: obj.getStart(),
+      end: obj.getEnd(),
```

Comment:
are you sure that this works? 
`obj` does not have any methods, it's just a data object read from JSON at this point.

based on the syntax you used, `this.start.valueOf()` and `this.end.valueOf()` might be needed

Reply:
Right, should have thought this through more thoroughly. 
Updated it.